tests/integration/conftest.py

```
@@ -161,6 +161,50 @@ def stub_nltk() -> Iterator[None]:
         yield
 
 
+@pytest.fixture()
```

Ruff (0.15.9)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)